File: LottieLib_test/app/src/main/java/com/daeseong/lottielib_test/Main4Activity.java
package com.daeseong.lottielib_test;

import androidx.appcompat.app.AppCompatActivity;
import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.os.Looper;
import android.util.Log;
import com.airbnb.lottie.LottieAnimationView;

public class Main4Activity extends AppCompatActivity {

    private static final String TAG = Main4Activity.class.getSimpleName();

    private LottieAnimationView lottieAnimationView;

    private Handler handler;
    private long lDelay;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main4);

        init();
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();

        clear();
    }

    private void init(){

        lottieAnimationView = (LottieAnimationView)findViewById(R.id.lav);
        lDelay = lottieAnimationView.getDuration() + 2000;

        handler = new Handler(Looper.getMainLooper());
        handler.postDelayed(new Runnable() {
            @Override
            public void run() {

                startActivity(new Intent(Main4Activity.this, MainActivity.class));
                finish();
            }
        }, lDelay);
    }

    private void clear(){

        try {

            if (lottieAnimationView != null) {
                lottieAnimationView.cancelAnimation();
            }

            if (handler != null) {
                handler.removeCallbacksAndMessages(null);
                handler = null;
            }

        }catch (Exception ex){
            Log.e(TAG, ex.getMessage());
        }
    }
}

File: lottielib_test_kotlin/app/src/main/java/com/daeseong/lottielib_test/Main4Activity.kt
package com.daeseong.lottielib_test

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity
import com.airbnb.lottie.LottieAnimationView

class Main4Activity : AppCompatActivity() {

    private val tag = Main4Activity::class.java.simpleName

    private lateinit var lottieAnimationView: LottieAnimationView

    private var handler: Handler? = null
    private var delay: Long = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main4)

        init()
    }

    override fun onDestroy() {
        super.onDestroy()

        clear()
    }

    private fun init() {

        lottieAnimationView = findViewById(R.id.lav)

        delay = lottieAnimationView.duration + 2000

        handler = Handler(mainLooper)
        handler?.postDelayed({
            startActivity(Intent(this@Main4Activity, MainActivity::class.java))
            finish()
        }, delay)
    }

    private fun clear() {

        try {
            lottieAnimationView.apply {
                cancelAnimation()
            }

            handler?.apply {
                removeCallbacksAndMessages(null)
                handler = null
            }
        } catch (ex: Exception) {
        }
    }
}